jobbergate-agent/jobbergate_agent/jobbergate/submit.py

```diff
@@ -45,6 +48,25 @@ def get_job_parameters(slurm_parameters: Dict[str, Any], **kwargs) -> SlurmJobPa
     return SlurmJobParams.parse_obj(merged_parameters)
 
 
+async def retrieve_job_script_file(job_script_file: JobScriptFile, submit_dir: Path):
```

Comment:
Nice touch. I suggest annotating the returning type as well.

Reply:
Done!